android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebView.java

```diff
@@ -15,6 +15,7 @@
 public class MagnetWebView extends WebView implements LifecycleEventListener {
     String TAG = "MagnetWebView";
     private boolean layoutSet;
+    public boolean loaded = false;
```

Comment:
I think adding a public property, that's not used by the owner is a hint that something's up.  See comment below.